File: Java/프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.java
import java.util.*;

class Solution {
    public int[] solution(String my_string) {
        int count = 0;
        
        for(int i=0; i<my_string.length(); i++) {
            char a = my_string.charAt(i);
            if(Character.isDigit(a)) {
                count++;
            }
        }
        
        int[] answer = new int[count];
        int index = 0;
        
        for(int i=0; i<my_string.length(); i++) {
            char a = my_string.charAt(i);
            if(Character.isDigit(a)) {
                answer[index] = a - '0';
                index++;
            }        
        }
        Arrays.sort(answer);
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.js
function solution(my_string) {
    var answer = [];
    for(let i=0; i<my_string.length; i++){
        if(my_string.charAt(i) == 1)
            answer.push(1);
        else if(my_string.charAt(i) == 2)
            answer.push(2);
         else if(my_string.charAt(i) == 3)
            answer.push(3);
         else if(my_string.charAt(i) == 4)
            answer.push(4);
         else if(my_string.charAt(i) == 5)
            answer.push(5);
         else if(my_string.charAt(i) == 6)
            answer.push(6);
         else if(my_string.charAt(i) == 7)
            answer.push(7);
         else if(my_string.charAt(i) == 8)
            answer.push(8);
         else if(my_string.charAt(i) == 9)
            answer.push(9);
         else if(my_string.charAt(i) == 0)
            answer.push(0);
    }
    answer.sort((a,b) => a-b);
    return answer;
}